### PR TITLE
Fix: custom slippage setting: check for NaN

### PR DESCRIPTION
**Pull request by jonator**
## Summary by CodeRabbit

- **Bug Fixes**
	- Improved number validation in slippage settings to ensure only valid numbers are accepted.
	- Enhanced boundary check to restrict manual slippage values within a specific range.

### CODE REVIEW

**Review by DavideSegullo — APPROVED** [review]

File: packages/stores/src/ui-config/slippage-config.ts [hunk 1]
```diff
@@ -64,15 +64,16 @@ export class ObservableSlippageConfig {
 
   @action
   setManualSlippage(str: string) {
+    if (isNaN(Number(str))) return;
```
Comment (JoseRFelix) [inline packages/stores/src/ui-config/slippage-config.ts:67]: Thoughts on using `isNumeric` instead?

Comment (jonator) [reply packages/stores/src/ui-config/slippage-config.ts:67]: Where can that be imported from?

Comment (jonator) [reply packages/stores/src/ui-config/slippage-config.ts:67]: Given this is the stores package, I don't have access to utils in web package.

Comment (JoseRFelix) [reply packages/stores/src/ui-config/slippage-config.ts:67]: This function is a good candidate for `@osmosis-labs/utils`. This suggestion is a nit, so feel free to merge or move it to the utils folder

**Pull request merged**